h/groups/templates/share.html.jinja2

```diff
@@ -18,6 +18,18 @@
       <div class="group-form__name-input">
         {{ group.name }}
       </div>
+
+      {% if document_links %}
+      Recently annotated documents:
+      <ul>
+        {% for document_link in document_links %}
+          <li>{{ document_link }}</li>
+        {% endfor %}
+      </ul>
+      {% else %}
+        <p>This group hasn't shared any annotations yet.</p>
```

Comment:
This sounds slightly odd. The group doesn't share annotations - people in the group do. "No annotations have been shared in this group yet"?